Portal/src/Datahub.Portal/Pages/Workspace/Dashboard/WorkspaceDashboard.razor

```diff
@@ -15,6 +15,17 @@
     return;
 }
 
+@if (_project.IsDeleted || _isDeleteRequested)
+{
+    <MudItem xs="12" sm="8">
+        <WorkspaceInfo WorkspaceAcronym="@WorkspaceAcronym" />
+    </MudItem>
+    <MudAlert Severity="Severity.Warning" Variant="Variant.Outlined" Style="color: #000;" Class="mb-6">
+        @Localizer["This workspace has been deleted"] 
+    </MudAlert>
```

Comment:
This is not mandatory but I just think the alert should be at the very top of the page
```suggestion
    <MudAlert Severity="Severity.Warning" Variant="Variant.Outlined" Style="color: #000;" Class="mb-6">
        @Localizer["This workspace has been deleted"] 
    </MudAlert>
    <MudItem xs="12" sm="8">
        <WorkspaceInfo WorkspaceAcronym="@WorkspaceAcronym" />
    </MudItem>    
```

Reply:
Discussed this change but ultimately felt it was not required.